### DISCUSSION TITLE
Solution Please ?

### DISCUSSION THREAD

**Discussion by CybroOdooDev** [flet-dev/flet#1795] [General]
https://github.com/flet-dev/serious-python/tree/main I tried the same in the mentioned link but I got an error:

Like this :
I/flutter ( 6225): Connecting to Flet server flet.sock...
I/flutter ( 6225): Connecting to Socket server flet.sock...
I/flutter ( 6225): Error connecting to Flet server: SocketException: Connection failed (OS Error: No such file or directory, errno = 2), address = flet.sock, port = 0
I/flutter ( 6225): Reconnect in 200 milliseconds

**Comment by phanstudio** [comment]
did you solve it
